.dockerignore

```diff
@@ -0,0 +1,3 @@
+test
+dist
+node_modules
```

Comment:
Since the git commands are not used in the Dockerfile, it would make sense to add the `.git` directory here.
This way the Docker cache will not be invalidated when the git commands are run on the developer machine